iolog.c

```diff
@@ -140,8 +140,13 @@ static int ipo_special(struct thread_data *td, struct io_piece *ipo)
 			break;
 		}
 		ret = td_io_open_file(td, f);
-		if (!ret)
+		if (!ret){
+			if (td->o.dp_type != FIO_DP_NONE) {
+				if (dp_init(td))
+					return false;
```

Comment:
If we return 0 here this fails silently. Why not print out an error message and abort on error? See the `td_verror()` + `return -1;` construction below.

Reply:
Following the construction below, I'll update it.